src/containers/ConversionForm/index.tsx

```diff
@@ -77,11 +77,20 @@ const ConversionForm = ({
 }: ConversionProps): JSX.Element => {
     const [fileToConvert, setFileToConvert] = useState<UploadFile | null>();
     const [isProcessing, setIsProcessing] = useState<boolean>(false);
-    const [serverErrorModalOpen, setServerErrorModalOpen] =
-        useState<boolean>(false);
-    const [fileTypeErrorModalOpen, setFileTypeErrorModalOpen] = useState(false);
+    const [conversionError, setConversionError] = useState<ConversionError>(
+        ConversionError.SERVER_ERROR
+    );
 
     const engineSelected = !!conversionProcessingData.engineType;
+    const errorModalOpen = conversionError !== ConversionError.NO_ERROR;
+
+    const errorMessage = {
+        [ConversionError.SERVER_ERROR]: "Please try again at a later time.",
+        [ConversionError.FILE_TYPE_ERROR]: `You may want to double check that the file you selected is a valid ${conversionProcessingData.engineType}
+                        file and try again.`,
```

Comment:
I'm seeing that this got broken up into two lines; will that introduce extra whitespace when this message is printed?

Reply:
No it looks ok on my machine